src/main/java/io/github/apace100/apoli/power/ActionOverTimePower.java

```diff
@@ -21,6 +22,9 @@
     private final Consumer<Entity> fallingAction;
 
     private boolean wasActive = false;
+    private boolean initialTicksSet = false;
+
+    private int initialTicks;
```

Comment:
Change type to `Integer` to allow a nullable value. `int` is a primitive type and shouldn't be un-set.
```suggestion
    private Integer initialTicks = null;
```